ui/src/components/startpage/Startpage.js

```diff
@@ -4,15 +4,15 @@ import { Card, CardContent, Grid } from "@material-ui/core/";
 import './startpage.css';
 import Upload from "../uploadERC/Upload";
 import InspectExamples from '../inspectExamples/InspectExamples';
-import config from '../../../helpers/config';
+//import config from '../../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.